daml-assistant/src/DA/Daml/Assistant/Install/Path.hs

```diff
@@ -135,7 +139,7 @@ prompt msg def others = do
 shellConfig :: String -> FilePath -> Maybe (FilePath, String)
 shellConfig shell targetPath =
   case shell of
-    "zsh" -> Just (".zshrc", "export PATH=$PATH:" <> targetPath)
-    "bash" -> Just (".bashrc", "export PATH=$PATH:" <> targetPath)
+    "zsh" -> Just (".profile", "export PATH=$PATH:" <> targetPath)
+    "bash" -> Just (".profile", "export PATH=$PATH:" <> targetPath)
```

Comment:
I think it's `.bash_profile` for bash and `.zprofile` for zsh.

Reply:
'.profile' works for bash officially and also seems to work for 'zsh' (I don't have a .zprofile but I do have a .profile)

Reply:
Side note / horror story: What happens if `targetPath` has spaces?

Reply:
but maybe this is called from with 'sh'. I changed it to '.bash_profile' and '.zprofile'. https://kb.iu.edu/d/abdy

Reply:
Given the abundance of possible files (`.bash_profile`, `.profile`, etc ...) maybe the algorithm should search for whichever is present, instead of hardcoding a single possible path for the profile?

E.g. I don't have any `.profile` but I have both `.bash_profile` and `.zprofile`.

Reply:
I think the mapping 'zsh -> .zprofile', 'bash -> .bash_profile' and 'sh -> .profile' is fairly standard and should cover the fast majority of setups.